src/test/java/Task3package/Task3.java

```diff
@@ -0,0 +1,20 @@
+package Task3package;
+
+
+public class Task3 {
+    public static void main(String[] args) {
+
+        Сhef c = new Сhef();
+
+        c.add("Man", "pink", 1, 5);
+        c.add("Tam", "yellow", 2, 6);
+        c.add("Son", "brown", 3, 7);
+        c.add("Ban", "black", 4, 8);
+        c.add("Ban", "black", 4, 8);
+        System.out.println("Vegetables equals : " + c.compareVegetables(c.getSalad().get(2), c.getSalad().get(4)));
+        c.DeleteByName("Ban");
+        c.DeleteByColor("yellow");
+        c.View();
+        System.out.println("Calories: " + c.getSaladCalories());
```

Comment:
you need to have:
- add vegetables to the salad
- remove vegetables from the salad
- display to the console a list of vegetables
- sort vegetables in the salad by calories
- sort vegetables in the salad by weight
- be able to compare two vegetables

make sure you have everything in place